amsclient/configuration.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2021 Red Hat, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package amsclient
+
+// Configuration represents the configuration of the AMS API client
+type Configuration struct {
+	Token    string `mapstructure:"token" toml:"token"`
+	URL      string `mapstructure:"url" toml:"url"`
+	PageSize int    `mapstrictire:"page_size" toml:"page_size"`
```

Comment:
+default page size perhaps? (your `100` constant above)

Reply:
Is it something that can be done in annotations? I will search for it, but I agree that, at least, I need to define a const for the default value